Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -59,6 +59,7 @@
 from L1Trigger.Configuration.L1Trigger_EventContent_cff import *
 from RecoVertex.BeamSpotProducer.BeamSpot_EventContent_cff import *
 from CommonTools.ParticleFlow.EITopPAG_EventContent_cff import EITopPAGEventContent
+from RecoCTPPS.TotemRPLocal.totemRPLocalReconstruction_cff import *
```

Comment:
if we do era-based settings in #14616, the same should be done for the rest of reco as well.